python/ray/exceptions.py

```diff
@@ -326,6 +326,28 @@ def __str__(self):
         )
 
 
+class OutOfDiskError(RayError):
+    """Indicates that the local disk is full.
+
+    This is raised if the attempt to store the object fails
+    because both the object store and disk are full.
+    """
+
+    def __str__(self):
+        # TODO(scv119): expose more disk usage information and link to a doc.
+        return super(OutOfDiskError, self).__str__() + (
+            "\n"
+            "The local object store is full and local disk is also full."
+            "Tip: Run `df` command to check the disk usage; "
+            "Use the `ray memory` command to check the plasma memory usage."
+            "You can adjust plasma memory capacity by calling "
+            "`ray.init(object_store_memory=...)`;"
+            "You can also try to spread out of disk usage across multiple "
+            "physical devices, following"
+            " https://docs.ray.io/en/master/ray-core/objects/object-spilling.html"
```

Comment:
I think this is too many action items. Also, we probably do not want to recommend changing the object_store_memory default (this is rarely a good idea for users).

```suggestion
            "Use `df` on this node to check disk usage and `ray memory` to check object store memory usage."
```